src/openrct2/ride/Vehicle.h

```diff
@@ -21,12 +21,10 @@
 #include "../world/map.h"
 
 #pragma pack(push, 1)
```

Comment:
This is not needed now then, based on your comment below. Furthermore, we only ever checked for size of `rct_ride_entry_vehicle` in with-RCT2 builds